ticket4uApp/t4uApp/utils.py

```diff
@@ -0,0 +1,99 @@
+import uuid
+from django.conf import settings
+import smtplib
+from email.message import EmailMessage
+from rest_framework.response import Response
+from rest_framework import status
+
+
+def make_bulk(num, obj):
+    bulk = []
+    counter = 1
+    while counter <= num:
+        bulk.append(obj)
+        counter += 1
+    return bulk
+
+
+def upload_to(instance, filename):
+    ext = filename.split('.')[-1]
+    filename = "%s.%s" % (uuid.uuid4(), ext)
+    # return 'images/{filename}'.format(filename=filename)
+    return filename
+
+
+def html_tickets(title, count):
+    return f'''
+    <div style="width: 100%; margin: 10px 0; border: 1px solid black; border-radius: 3px; display: flex; padding: 10px; height: 220px">
```

Comment:
the same about inline styles looks misunderstanding, you can read about BEM https://en.bem.info/methodology/